examples/client/client.c

```diff
@@ -3007,6 +3007,16 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
 
     showPeerEx(ssl, lng_index);
 
+#ifdef HAVE_OCSP
+    {
+        struct tm tm;
+        char date[32];
```

Comment:
Is there a better place to get the max size 32 bytes from?

Reply:
That "32" is a function of the format string in L3015, so nowhere to get it from.  Kinda lame.  It sure would be nice if there were a standard header that defined `ISO8601_DATE_FORMAT` and `_SIZE`.  Hmm maybe we should add it to one of our headers.